assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/variables.tf

```diff
@@ -85,7 +85,6 @@ variable "asset_dir" {
 variable "network_mtu" {
   description = "CNI interface MTU"
   type        = number
-  default     = 1480
```

Comment:
Why is the default value removed ?

Reply:
So we're able to modify it from the template to account for wireguard overhead.